doc/19_Development_Tools_and_Details/08_Generic_Execution_Engine/README.md

```diff
@@ -1,23 +1,99 @@
 # Generic Execution Engine
-:::caution
 
-To use this feature, please update your `composer.json` by requiring `phpdocumentor/reflection-docblock` and `symfony/property-info` like in the `suggest` section. Enable the `PimcoreGenericExecutionEngineBundle` in your `bundles.php` file and install it accordingly with the following command:
+## Overview
 
+The Generic Execution Engine provides:
+* Asynchronous job execution via Symfony Messenger
+* State tracing and logging for job runs  
+* Job run management (start, cancel, restart)
 
-`bin/console pimcore:bundle:install PimcoreGenericExecutionEngineBundle`
+## Installation
 
+1. Update your `composer.json` to require the necessary dependencies:
+   ```bash
+   composer require phpdocumentor/reflection-docblock symfony/property-info
+   ```
+
+2. Enable the `PimcoreGenericExecutionEngineBundle` in your `/config/bundles.php` file by adding this line:
+   ```php
+   Pimcore\Bundle\GenericExecutionEngineBundle\PimcoreGenericExecutionEngineBundle::class => ['all' => true],
+   ```
+
+3. Install the bundle:
+   ```bash
+   bin/console pimcore:bundle:install PimcoreGenericExecutionEngineBundle
+   ```
+
+## Essential Configuration
+
+### Step 1: Configure Messenger Transport
+
+:::caution Transport Configuration Required
+The execution engine **requires** the `pimcore_generic_execution_engine` transport to be configured. Without this, jobs will not be processed.
+:::
+
+Add this transport to your existing configuration:
+
+```yaml title="messenger.yaml"
+framework:
+    messenger:
+        transports:
+            # ... your existing transports
+            pimcore_generic_execution_engine: 'amqp://rabbitmq:5672/%2f/pimcore_generic_execution_engine'
```

Comment:
isn't that an optional thing? 
the transport should be configured by default: see https://github.com/pimcore/pimcore/blob/12.x/bundles/GenericExecutionEngineBundle/config/pimcore/config.yaml#L19

only thing which is important is to configure the message worker consuming the transport...

Reply:
Sorry for being a bit picky now :smile:
What about changing the order of consumption and transport. First write, what is mandatory to do (configure the consumption), and then add optional steps. 
I'm afraid users stop when reading optional ... and never get to the mandatory steps.